### PR TITLE
fix(ci): prepare base tag to use new convention

### PR DESCRIPTION
**Pull request by peturgq**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by brynjarorng — APPROVED** [review]
LGTM

File: scripts/ci/00_prepare-base-tags.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ if [[ "$BUILD_REF" != "$LAST_GOOD_BUILD_SHA" ]]; then
   git log -1 "$BUILD_REF"
 fi
 LAST_GOOD_BUILD_DOCKER_BRANCH_TAG=$(echo "${LAST_GOOD_BUILD_BRANCH}" | tr "/." "-" )
-export LAST_GOOD_BUILD_DOCKER_TAG=${LAST_GOOD_BUILD_RUN_NUMBER}_${LAST_GOOD_BUILD_DOCKER_BRANCH_TAG}_${LAST_GOOD_BUILD_SHA:0:7}
+export LAST_GOOD_BUILD_DOCKER_TAG=${LAST_GOOD_BUILD_DOCKER_BRANCH_TAG}_${LAST_GOOD_BUILD_SHA:0:7}_${LAST_GOOD_BUILD_RUN_NUMBER}
```
Comment (busla) [inline scripts/ci/00_prepare-base-tags.sh:22]: have you tested this locally?

**Review by busla — CHANGES_REQUESTED** [review]
LGTM

File: scripts/ci/00_prepare-base-tags.sh (hunk 1 above)
Comment (peturgq) [reply scripts/ci/00_prepare-base-tags.sh:22]: Yes
```    ~/git/island-is/island.is/scripts/ci    main *20    ./00_prepare-base-tags.sh                                                               ✔ 
+++ dirname ./00_prepare-base-tags.sh
++ cd .
++ pwd
+ DIR=/home/petur/git/island-is/island.is/scripts/ci
+ ROOT=/home/petur/git/island-is/island.is/scripts/ci/../..
++ mktemp -d
+ tempRepo=/tmp/tmp.nnrkr3xTuG
+ cp -r /home/petur/git/island-is/island.is/scripts/ci/../../.github/actions/dist/. /tmp/tmp.nnrkr3xTuG
++ DEBUG='*,-simple-git'
++ REPO_ROOT=/home/petur/git/island-is/island.is/scripts/ci/../..
++ node /tmp/tmp.nnrkr3xTuG/main.js
/tmp/tmp.nnrkr3xTuG/main.js:24093
        throw new Error("[@octokit/auth-action] `GITHUB_ACTION` environment variable is not set. @octokit/auth-action is meant to be used in GitHub Actions only.");
              ^

Error: [@octokit/auth-action] `GITHUB_ACTION` environment variable is not set. @octokit/auth-action is meant to be used in GitHub Actions only.```

**Pull request merged**